### PR TITLE
Search CRUD topics - part 2

### PR DESCRIPTION
**Pull request by jamesm-macrometa**
Part 2 of Search CRUD topics update.  This one includes Rename and Delete a search view.

Note for reviewers: The JavaScript "Rename a Search View" sample does not work 100% as intended as it returns `undefined` instead of the new Search View name.  I could use some help fixing this since my JS is not nearly as strong as my Python.

### CODE REVIEW

**Comment by jamesm-macrometa** [comment]
@koshy-thomas @martin-macrometa Can you please review the JavaScript and Python samples for "Rename a Search View" and "Delete a Search View"?  They work for me, but I'd feel better with engineer's blessing before we go live.  😃 

Additionally, I need help troubleshooting the JavaScript sample for "Rename a Search View."  For some reason it always returns `undefined` instead of the new Search View name.  However, I got it to work fine with Python.

File: docs/search/views/rename-search-view.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+```
+
+</TabItem>
+<TabItem value="js" label="JavaScript SDK">
+
+```js
+// Connect to GDN.
+const jsc8 = require("jsc8");
+const client = new jsc8({url: "https://play.paas.macrometa.io", apiKey: "TD0IqyKNnQaq5F88bRg0GAg.james_test2.CnC1oHcyDucQRJdoK0EoalovMDGAV1LqWjxI0dPO7PlLfyXza3dq9PfT3CIXEFz4d60ced", fabricName: "_system"});
```
Comment (koshy-thomas) [inline docs/search/views/rename-search-view.md]: @jamesm-macrometa it seems you left your apikey here

File: docs/search/views/rename-search-view.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+async function renameMySearchView () {
+  const listOfViews = await client.getListOfViews();
+  const view = listOfViews.result.find(v => v.name === searchViewName);
+  if (!view) {
+    console.log(`Search view "${searchViewName}" does not exist`);
+    return;
+  }
+  const renamedView = await client.renameView(view.id, newSearchViewName);
+  console.log(`Successfully renamed search view: ${renamedView.name}`);
```
Comment (koshy-thomas) [inline docs/search/views/rename-search-view.md]:
```suggestion
  console.log(`Successfully renamed search view: ${renamedView.result.name}`);
```

Comment (koshy-thomas) [reply docs/search/views/rename-search-view.md]: It should work now after adding the above comments

File: docs/search/views/rename-search-view.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+const newSearchViewName = "NewSearchView"; // Replace with the new name for the search view.
+
+async function renameMySearchView () {
+  const listOfViews = await client.getListOfViews();
+  const view = listOfViews.result.find(v => v.name === searchViewName);
+  if (!view) {
+    console.log(`Search view "${searchViewName}" does not exist`);
+    return;
+  }
+  const renamedView = await client.renameView(view.id, newSearchViewName);
```
Comment (koshy-thomas) [inline docs/search/views/rename-search-view.md]:
```suggestion
  const renamedView = await client.renameView(view.name, newSearchViewName);
```

File: docs/search/views/delete-search-view.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+const searchViewName = "Search"; // Replace with the name of the search view you want to delete.
+
+async function deleteMySearchView() {
+  const listOfViews = await client.getListOfViews();
+  const view = listOfViews.result.find(v => v.name === searchViewName);
+  if (!view) {
+    console.log(`Search view "${searchViewName}" does not exist`);
+    return;
+  }
+  await client.deleteView(view.id);
```
Comment (koshy-thomas) [inline docs/search/views/delete-search-view.md]:
```suggestion
  await client.deleteView(view.name);
```

Comment (koshy-thomas) [reply docs/search/views/delete-search-view.md]: let's delete based on name rather than id.

File: docs/search/views/rename-search-view.md (hunk 3 above)
Comment (koshy-thomas) [reply docs/search/views/rename-search-view.md]: let's rename based on name rather than id.

**Review by martin-macrometa — APPROVED** [review]

File: docs/search/views/rename-search-view.md (hunk 3 above)
Comment (koshy-thomas) [reply docs/search/views/rename-search-view.md]: @jamesm-macrometa it seems it again got changed to "id" somehow

Comment (koshy-thomas) [reply docs/search/views/rename-search-view.md]: I am just committing it again and resolving this comment

**Review by koshy-thomas — APPROVED** [review]

File: docs/search/views/rename-search-view.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    console.log(`Search view "${searchViewName}" does not exist`);
+    return;
+  }
+  const renamedView = await client.renameView(view.name, newSearchViewName);
+  console.log(`Successfully renamed search view: ${renamedView.result.name}`);
+}
+
+renameMySearchView();
+
+
```
Comment (diana-macrometa) [inline docs/search/views/rename-search-view.md]:
```suggestion
```

**Review by diana-macrometa — APPROVED** [review]
One minor edit, then good to go!

**Pull request merged**